ovos_plugin_manager/templates/persona.py

```diff
@@ -0,0 +1,44 @@
+from ovos_bus_client.message import dig_for_message
+from ovos_utils import classproperty
+from ovos_utils.process_utils import RuntimeRequirements
+from quebra_frases import span_indexed_word_tokenize, word_tokenize
+
+
+class Persona:
```

Comment:
no class needed, personas are .json files

https://github.com/OpenVoiceOS/ovos-persona/tree/dev/personas

Reply:
I'm not sure I understand the entrypoints here then; if its just configuration, what would a persona package be providing? Feel free to push to this branch directly

Reply:
user provided personas are .json files

package provided personas are a dict, which OPM can then scan, literally just need to delete this class